docs/tasks.md

```diff
@@ -0,0 +1,143 @@
+Similarly, You can create a build task from the VS Code command pallette by running the **Tasks: Configure task** command.
+
+By selecting "CMake: build" template, if you are not using presets, this task will be generated in tasks.json file: 
+
+```json
+    {
+        "type": "cmake",
+        "label": "CMake: build",
+        "command": "build",
+        "targets": [
+            "all"
+        ],
+        "group": "build",
+        "problemMatcher": [],
+        "detail": "CMake template build task"
+    }
+```
+Note: When running this task, the build settings (including buildArgs and build environment) defined in settings.json will be used.
+
+However, if you are using presets, this task will be generated in tasks.json file:
+
+```json
+    {
+        "type": "cmake",
+        "label": "CMake: build",
+        "command": "build",
+        "preset": "${command:cmake.activeBuildPresetName}",
+        "detail": "CMake template build task"
+    }
+```
+
+Note: When running this task, the configure settings defined in CMakeUserPresets.json/CMakePresets.json will be used.
+
+You can chain a configure task to your build task by adding this to your task's definition:
+
+```json
+        "dependsOn": [
+            "CMake: configure"
+        ]
+```
+
+# Install with CMake Tools tasks
+Similarly, You can create an install task from the VS Code command pallette by running the **Tasks: Configure task** command.
+
+By selecting "CMake: install" template, this task will be generated in tasks.json file:
+
+```json
+    {
+        "type": "cmake",
+        "label": "CMake: install",
+        "command": "install",
+        "problemMatcher": [],
+        "detail": "CMake template install task"
+    }
+```
+
+# Test with CMake Tools tasks
+Similarly, You can create a test task from the VS Code command pallette by running the **Tasks: Configure task** command.
+
+By selecting "CMake: test" template, if you are not using presets, this task will be generated in tasks.json file: 
+
+```json
+    {
+        "type": "cmake",
+        "label": "CMake: test",
+        "command": "test",
+        "detail": "CMake template test task"
+    }
+```
+Note: When running this task, the test settings defined in settings.json will be used.
+
+However, if you are using presets, this task will be generated in tasks.json file:
+
+```json
+    {
+        "type": "cmake",
+        "label": "CMake: test",
+        "command": "test",
+        "preset": "${command:cmake.activeTestPresetName}",
+        "detail": "CMake template test task"
+    }
+```
+
+Note: When running this task, the test settings defined in CMakeUserPresets.json/CMakePresets.json will be used.
+
+# Clean/Clean-rebuild with CMake Tools tasks
+Similarly, you can create a Clean/Clean-rebuild task from the VS Code command pallette by running the **Tasks: Configure task** command.
+
+By selecting "CMake: clean"/"CMake: clean rebuild" template, these task will be generated in tasks.json file:
+
+```json
+		{
```

Comment:
Please remove some of the indentation here to match the other sections in the doc.